_pages/dev/tag-management-system.md

```diff
@@ -50,25 +48,22 @@ However, the process _usually_ requires you to specify a certain `<script>` tag
 <!-- End Google Tag Manager -->
 ```
 
-After this is done, we can move on to the [configuration](#configuration) section.
-
-For more example, please check our [TMS examples doc](tag-management-system-examples.md).
+For see additional setup examples with Google Analytics and AELP, see [Tag Management System Examples]({{ site.baseurl }}{% link _pages/dev/tag-management-system-examples.md %}).
 
 ## Configuration
 
 The following TMS configuration options are available:
 
-- `debug?: boolean` - Should be enabled in development mode only. Enables console logs.
-- `dataLayerProperty?: string` - The name for the data layer object. Provide only if you are using a different name than the default one with the default data-layer data structure (e.g. for `gtm` it's an array-like, while for `aelp` it's an empty object).
-- `events?: AbstractType<CxEvent>[]` - List all the event classes which are to be collected and pushed to the data layer.
-- `collector?: Type<TmsCollector>` - The custom collector service implementation.
-- `gtmId` - Available only for GTM. By providing it, Spartacus will handle the script "injection" and bootstrapping for you.
-- `scriptUrl` - Available only for AELP. By providing it, Spartacus will handle the script "injection" and bootstrapping for you.
+- `debug` is a boolean that enables the console log. It should only be enabled in development mode.
+- `dataLayerProperty` is a string that names the data layer object. You only need to provide the `dataLayerProperty` if you are not using the default name for the default data layer data structure. In the case of GTM, the default data structure for the data layer is an array-like object, and in the case of AELP, it is an empty object.
```

Comment:
Hmm, maybe we can simplify this by saying: "is a string that names the data layer object. You only need to provide the `dataLayerProperty` if you are not using the default name for the data layer".

Reply:
Excellent! 👍 🙂